app/presenters/content_item/metadata.rb

```diff
@@ -20,12 +20,22 @@ def important_metadata
     end
 
     def publisher_metadata
-      {
+      display_date = details_display_date
+      metadata = {
         from:,
         first_published: published,
         last_updated: updated,
-        see_updates_link: true,
       }
+
+      unless display_date.present? && Time.zone.parse(display_date).future?
```

Comment:
We could put this in a method, `pending_stats_announcement?` so it's easier to follow what's going on?